kernel/src/main/java/org/kframework/kast/KastFrontEnd.java

```diff
@@ -114,7 +114,7 @@ public int run() {
                 parsed = new ExpandMacros(compiledMod, files, def.kompileOptions, false).expand(parsed);
             }
             if (options.kore) {
-              ModuleToKORE converter = new ModuleToKORE(compiledMod, files, def.topCellInitializer);
+              ModuleToKORE converter = new ModuleToKORE(compiledMod, files, null);
```

Comment:
why was this done?

Reply:
Sorry about that!  I think it happened while doing conflict resolution during the merge; for some reason I didn't noticed I can use `def.topCellInitializer` there.  I'm reverting the change.